File: Problem Solve/Programmers/[Lv.1] 로또의 최고 순위와 최저 순위/solution.swift
import Foundation

func solution(_ lottos:[Int], _ win_nums:[Int]) -> [Int] {
    let doodle = lottos.filter { $0 == 0 }.count
    let correctNum = lottos.filter { win_nums.contains($0) }.count
    
    var min = 6 - correctNum + 1
    var max = min - doodle
    
    if min > 6 {
        min = 6
    }
    if max > 6 {
        max = 6
    }
    
    return [max, min]
}

let lottos = [2, 3, 4, 100, 0, 1900]
let win_nums = [31, 10, 45, 1, 6, 19]

print(solution(lottos, win_nums))

File: Problem Solve/Programmers/[Lv.1] 로또의 최고 순위와 최저 순위/solution.py
def solution(lottos, win_nums):
    answer = []
    check = 0
    masterKey = lottos.count(0)

    for i in range(0, 6):
        for j in range(0, 6):
            if(lottos[i] == win_nums[j]):
                check += 1

    highest = 7 - (check + masterKey)
    lowest = 7 - check
    if highest == 7:
        highest = 6
    if lowest == 7:
        lowest = 6

    answer.append(highest)
    answer.append(lowest)

    return answer
